enterprise/metrics/src/main/java/org/neo4j/metrics/source/DBMetrics.java

```diff
@@ -195,6 +210,50 @@ public Long getValue()
             } );
         }
 
+        // check pointing
+        if ( config.get( MetricsSettings.neoCheckPointingEnabled ) )
```

Comment:
Method `start()` is quite big now. Could we please extract all `Gauge` setup into separate methods like `registerTxGauges`, `registerCheckPointGauges`, etc. ?